news/2018-05-26-beatmap-spotlights-april-2018.md

```diff
@@ -0,0 +1,144 @@
+
+[![](/wiki/shared/news/2018-05-26-beatmap-spotlights-april-2018/jojo.jpg)](http://osu.ppy.sh/beatmapsets/714329)
+
+[Delis](https://osu.ppy.sh/users/1603923) is already well-known as the resident spotlight hogger, but for good reasons! Sono Chi no Sadame is a really intense and epic song, and Delis' representation of the map was done impeccably. High and low points of the song were captured perfectly with variations on spacing and rhythms, and all not without a touch of the classic Delis style™. A solid addition to this month's Spotlight!
+
+## osu!taiko
+
+[![](/wiki/shared/news/2018-05-26-beatmap-spotlights-april-2018/world-fragments.jpg)](http://osu.ppy.sh/beatmapsets/721551)
+
+While certainly not a stranger to the Spotlights, [iloveyou4ever](https://osu.ppy.sh/users/4964596) has outdone himself this time with his map of xi's World Fragments! This ten-minute marathon takes the player through three movements of music, each of which contains everything from sweeping piano passages to calm melodic bits.
+
+What makes this map unique, however, is the way iloveyou4ever approached the song. Every section is distinctly patterned to meld into xi's composition, yet the map still flows cohesively as a whole. This is a selection worth playing through for the experience -- ten minutes will fly by in an instant!
+
+[![](/wiki/shared/news/2018-05-26-beatmap-spotlights-april-2018/nexta.jpg)](http://osu.ppy.sh/beatmapsets/734616)
+
+After the slamming success of both "Cycle Hit" and "We Are Number One" back in 2017, [Midnaait](https://osu.ppy.sh/users/1506011) is back again this year with their next(a) hit, lapix's "Nexta"!
+
+Being the winner of the o!Taiko Argentina Mapping Contest, it is no surprise that this gem is included in this month's Spotlights. Featuring several tricky patterns throughout the entirety of the "Data Overwrite" difficulty mixed with SV changes and a brutal long stream at the end, this map will definitely catch any unsuspecting players off-guard.
+
+Not ready for 160BPM streams? No worries, players of all levels will find something to love in this set as there are a total of 5 beautifully crafted difficulties to choose from.
+
+[![](/wiki/shared/news/2018-05-26-beatmap-spotlights-april-2018/strike.jpg)](http://osu.ppy.sh/beatmapsets/713086)
+
+An excellent mapset by [frukoyurdakul](https://osu.ppy.sh/users/7612550) who extremely familiar with rock songs. Every difficulties are well mapped especially hardest one 'K.O'. He shows us what's rock song's essence.The triple and stream emphasize guitar, drum and even vocal perfectly. Moreover, the hitsound usage of stream is quite amazing and feels good while playing. If you wants to play 1/3+1/4 stuff, this map might be a good choice.
```

Comment:
essence.The triple and stream emphasize guitar
essence. The triples and streams emphasize the guitar

Reply:
just double check the taiko writeups